deploy/neon/templates/scripts.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: neon-compute-scripts
+  labels:
+    {{- include "neon.labels" . | nindent 4 }}
+data:
+  compute.sh: |
```

Comment:
Like https://github.com/apecloud/kubeblocks/blob/0dea7c6b9f7b58d0d09e2d754810cfed3e163f5e/deploy/apecloud-mysql/templates/scripts.yaml#L8-L8, extra all scripts to scripts directory.